code/modules/projectiles/projectile.dm

```diff
@@ -97,6 +97,11 @@
 				splatter_dir = get_dir(starting, target_loca)
 			if(isalien(L))
 				new /obj/effect/temp_visual/dir_setting/bloodsplatter/xenosplatter(target_loca, splatter_dir)
+			if(isipc(L)) // Not the right solution. I need it to check if the location hit is BODYPART_ROBOTIC, but I don't know how to do that.

```

Comment:
var/obj/bodypart/B = L.get_bodypart(def_zone)
if(B.status == ORGAN_ROBOTIC)

or something close to that.